src/System.Buffers/tests/ArrayPool/UnitTests.cs

```diff
@@ -429,7 +429,7 @@ public static void ReturningANonPooledBufferOfDifferentSizeToThePoolThrows()
         {
             ArrayPool<byte> pool = ArrayPool<byte>.Create(maxArrayLength: 16, maxArraysPerBucket: 1);
             byte[] buffer = pool.Rent(15);
-            Assert.Throws<ArgumentException>("array", () => pool.Return(new byte[1]));
+            Assert.Throws<ArgumentException>(null, () => pool.Return(new byte[1]));
```

Comment:
@weshaggard, @stephentoub Any idea on why the argument name wouldn't surface? The string is definitely in the implementation in CoreLib.